### PR TITLE
Fixing high-contrast issues

### LINKED ISSUE #1903: SearchBox on high contrast mode should show dotted border on focus
Currently on **high contrast mode**, the SearchBox component doesn't show a dotted border on it when it has focus. That would be nice to have for added accessibility.

### PR DESCRIPTION
**Pull request by kristoferbrown**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #1903, #2656, #2766, #2858, #2854, #2781
- [x] Include a change request file using `$ npm run change`

#### Description of changes

High contrast fixes for breadcrumb, contextualMenu, overlay, pivot, calendar, and searchbox

#### Focus areas to test

breadcrumb, contextualMenu, overlay, pivot, calendar, and searchbox

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss [hunk 1]
```diff
@@ -153,7 +154,7 @@ $Calendar-dayPicker-margin: 10px;
 }
 
 .day.dayIsHighlighted.dayIsFocused {
-  @include focus-border(1px, $ms-color-white);
+  @include focus-border(1px, WindowText);
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: hmm. This seems like a hardcoded color that won't get replaced by theming. I'm not sure that's correct. Shouldn't this be scoped within a hc selector?

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: it seems like we may be trading one bug for another here, careful

**Comment by kristoferbrown** [comment]
@dzearing, great catch. I've fixed it so that the system color in calendar is scoped inside a HC selector.

@lorejoh12 and @johannao76, I would appreciate your review as the calendar code owner.

**Comment by micahgodbolt** [comment]
I think the calendar changes are tiny enough to not worry about them. Is this ready to merge @kristoferbrown ?

**Pull request merged**